### PR TITLE
New Package: OpenTX.OpenTXCompanion version 2.3.15

### PR DESCRIPTION
**Pull request by a-mnich**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113355&drop=dogfoodAlpha

### CODE REVIEW

**Comment by a-mnich** [comment]
The download links are provided at the bottom of the [release notes](https://www.open-tx.org/2022/04/22/opentx-2.3.15).  
I also put a link to the release notes in the manifest.

File: manifests/o/OpenTX/OpenTXCompanion/2.3.15/OpenTX.OpenTXCompanion.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: OpenTX.OpenTXCompanion
+PackageVersion: 2.3.15
+PackageLocale: en-US
+Publisher: OpenTX
+PublisherUrl: https://www.open-tx.org/
+PublisherSupportUrl: https://github.com/opentx/opentx/issues
+PackageName: OpenTXCompanion
+License: GPL-2.0
```
Comment (stephengillie) [inline manifests/o/OpenTX/OpenTXCompanion/2.3.15/OpenTX.OpenTXCompanion.locale.en-US.yaml:11]:
```suggestion
PackageUrl: https://www.open-tx.org/2022/04/22/opentx-2.3.15
License: GPL-2.0
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**